homework.md

```diff
@@ -1,113 +1,178 @@
 # Linus's Homework
 
+Name:Ali Nosratipour
+City: London
+Slack Name:london-class7
+
+Homework Details
+Module:Git/Github
+Week: 2
+
 ## Question 1
 
 What is 2 + 2?
 
 5
+correct answer is: 4
 
 ## Question 2
 
 What is the opposite of Black?
 
 Purple
+correct answer is: White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
 
 1322
+correct answer is: 12,240
 
-## Question 4 
+## Question 4
 
 What does HTML stand for?
 
 Hyper Tall Massive Language
 
+correct answer is: Hyper Text Markup Language
+
 ## Question 5
 
 What is CSS useful for?
 
 Making a website go faster
 
+correct answer is: making website look good
+
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
 #h1 {
-
 }
 ```
 
+correct answer is: h1{};
+
 ## Question 7
 
 What is JavaScript?
 
 A programming language used on the internet
 
+correct answer is:
+
+JavaScript is the Programming Language for the Web.
+it can update and change both HTML and CSS.
+it can calculate, manipulate and validate data.
+now days you can even use javascript to develop desktop app
+
 ## Question 8
 
 What three problem does Git & Github solve?
 
 When people want to show off code to each other they can put it on Github
 
+correct answer is:
+
+Documentation.
+Showcase your work.
+Markdown.  
+Track changes in your code across versions.
+Integration options.
+Collaboration
+
 ## Question 9
 
 What happens when you `fork` a repository?
 
 You delete it
 
-## Question 10 
+correct answer is:
+A fork is a copy of a repository. Forking a repository allows you to freely experiment with changes without affecting the original project.
+
+## Question 10
 
 What happens when you clone a repostory?
 
 It send it to a friend
 
+correct answer is:
+
+The "clone" command downloads an existing Git repository to your local computer. You will then have a full-blown, local version of that Git repo and can start working on the project.
+
 ## Question 11
 
 What is a Pull Request?
 
 When you send a file over the internet
 
+correct answer is:
+
+Pull Requests are the heart of collaboration on GitHub. When you open a pull request, you're proposing your changes and requesting that someone review and pull in your contribution and merge them into their branch.
+
 ## Question 12
 
 What is the Git History log?
 
 It lets you see into the future
 
+correct answer is:
+History log keeps record of commits
+
 ## Question 13
 
 What does it meant to "push" changes to Github?
 
 You send the changes of your local repository to Github
 
+correct answer is:
+
+when you want to upload your project or files to github repository.
+
 ## Question 14
 
 What is Github?
 
 A place for us to store and share our code.
 
+correct answer is:
+
+A website to create and hold repository and it will sync with git software
```

Comment:
Hi Ali, I don't think github Sync with git necessarily

Reply:
I'm not sure but I think make it sync according to this link but I will ask it from teacher to make sure https://docs.github.com/en/free-pro-team@latest/desktop/contributing-and-collaborating-using-github-desktop/keeping-your-local-repository-in-sync-with-github